### PR TITLE
added metadata to spans and strips example

### PR DESCRIPTION
**Pull request by AkintolaRichard**
- [x] issues: fixes #11765 
- [x] tests added / passed

### CODE REVIEW

File: examples/plotting/spans_and_strips.py [hunk 1]
```diff
@@ -1,3 +1,12 @@
+''' An example demonstration the use of `hspan`, `wspan`, and `vstrip`.
```
Comment (bryevdv) [inline examples/plotting/spans_and_strips.py]: typo on `wspan`, and ReST formatting actually requires *double* backticks. Also this phrasing would be more consistent with existing other examples:

```suggestion
''' This example demonstrates the use of ``hspan``, ``vspan``, ``hstrip``, and ``vstrip``.
```

File: examples/plotting/spans_and_strips.py [hunk 2]
```diff
@@ -1,3 +1,12 @@
...
+It also uses the `HoverTool`.
```
Comment (bryevdv) [inline examples/plotting/spans_and_strips.py]: I'd suggest elaborating at least a little, maybe:
```suggestion
It also demonstrates how the default ``HoverTool`` interacts with these glyphs.
```

File: examples/plotting/spans_and_strips.py [hunk 3]
```diff
@@ -1,3 +1,12 @@
...
+
+.. bokeh-example-metadata::
+    :apis: bokeh.models.HoverTool
+    :refs: :ref:`ug_basic_hspan`
```
Comment (bryevdv) [inline examples/plotting/spans_and_strips.py]: This reference does no exist, as far as I know. I don't think think there are currently direct refs for these, so maybe jsut delete this line entirely. cc @tcmetzger for any other suggestions

File: examples/plotting/spans_and_strips.py [hunk 4]
```diff
@@ -1,3 +1,12 @@
+''' An example demonstration the use of `hspan`, `wspan`, and `vstrip`.
+It also uses the `HoverTool`.
+
+.. bokeh-example-metadata::
+    :apis: bokeh.models.HoverTool
```
Comment (bryevdv) [inline examples/plotting/spans_and_strips.py]: The hover tool is secondary for this example, I'd suggest refs for `bokeh.plotting.figure.hspan` and the other glyphs instead.

**Review by bryevdv — COMMENTED** [review]
Thanks for the PR @AkintolaRichard ! A few comments to share

**Comment by AkintolaRichard** [comment]
Thank you for the review @bryevdv , I will effect the changes and get back to you.

**Comment by AkintolaRichard** [comment]
I have made the changes, please do check it out @bryevdv. Do you mind if I rebase it?

**Comment by bryevdv** [comment]
@AkintolaRichard rebase is fine! FYI I am traveling the next five days but I'll look at things when I return (unless @tcmetzger  can look sooner)

**Comment by AkintolaRichard** [comment]
Thank very much @bryevdv, I will rebase and await your return. I wish you a lovely journey experience.

**Comment by AkintolaRichard** [comment]
I would create a new PR for this issue with same changes, so this particular one can be deleted, since it couldn't pass the test due to merge conflicts.

**Comment by AkintolaRichard** [comment]
I just made a new PR that will replace this particular PR, here is the new PR id #13704.

File: examples/plotting/spans_and_strips.py [hunk 5]
```diff
@@ -1,3 +1,11 @@
+''' This example demonstrates the use of ``hspan``, ``wspan``, and ``vstrip``.
```
Comment (bryevdv) [inline examples/plotting/spans_and_strips.py]: still a typo here "wspan"

File: examples/plotting/spans_and_strips.py [hunk 6]
```diff
@@ -1,3 +1,11 @@
...
+It also demonstrates how the default ``HoverTool`` interacts with these glphs.
```
Comment (bryevdv) [inline examples/plotting/spans_and_strips.py]: also typo here "glphs" - > "glyphs"

File: examples/plotting/spans_and_strips.py [hunk 7]
```diff
@@ -1,3 +1,11 @@
...
+
+.. bokeh-example-metadata::
+    :apis: bokeh.plotting.figure.hspan, bokeh.plotting.figure.hstrip, bokeh.plotting.figure.vstrip
+    :keywords: strips, hover tool
+'''
```
Comment (bryevdv) [inline examples/plotting/spans_and_strips.py]: We try to end docstrings with a blank line
```suggestion
    :keywords: strips, hover tool
    
'''
```

**Comment by bryevdv** [comment]
HI @AkintolaRichard stil a couple of issues to resolve, noted above. Also the codebase failure is unrelated, merging in the latest `branch-3.4` should fix them (I think)

File: examples/plotting/spans_and_strips.py (hunk 5 above)
Comment (AkintolaRichard) [reply examples/plotting/spans_and_strips.py]: Oh my, I was literally reading it out loud when I was making the changes, I guess my hands was faster than my mouth. I will effect the changes.

File: examples/plotting/spans_and_strips.py (hunk 6 above)
Comment (AkintolaRichard) [reply examples/plotting/spans_and_strips.py]: I will make the correction.

File: examples/plotting/spans_and_strips.py (hunk 7 above)
Comment (AkintolaRichard) [reply examples/plotting/spans_and_strips.py]: I will add the newline.

**Comment by AkintolaRichard** [comment]
Thank you for your patience @bryevdv, I do appreciate the review. I will make the highlighted changes required and I will get back to you. Thank you also for the clarification on the test failures, I was worried about them.

**Comment by AkintolaRichard** [comment]
I have made the changes highlighted, please do check it out @bryevdv.

File: examples/plotting/spans_and_strips.py [hunk 8]
```diff
@@ -1,3 +1,13 @@
+''' This example demonstrates the use of ``hspan``, ``vspan``, ``hstrip`` and
+``vstrip``.
+It also demonstrates how the default ``HoverTool`` interacts with these glyphs.
```
Comment (bryevdv) [inline examples/plotting/spans_and_strips.py]: no need for a newline between sentences
```suggestion
``vstrip``. It also demonstrates how the default ``HoverTool`` interacts with these glyphs.
```

**Review by bryevdv — COMMENTED** [review]
Thanks @AkintolaRichard one small comment

File: examples/plotting/spans_and_strips.py (hunk 8 above)
Comment (bryevdv) [reply examples/plotting/spans_and_strips.py]: I'm going to go ahead and commit this tiny change for the sake of getting this merged

**Comment by bryevdv** [comment]
Thanks @AkintolaRichard

**Comment by AkintolaRichard** [comment]
Thank you @bryevdv, I do appreciate.

**Pull request merged**